### PR TITLE
Update popup.html with some sweet new ones. Namely, the endless crying.

### PR DESCRIPTION
**Pull request by knewkirk**
Because you can never have too much ascii art.

### CODE REVIEW

File: popup.html [hunk 1]
```diff
@@ -8,9 +8,11 @@
     <ul>
         <li>ಠ_ಠ</li>
         <li>(╯°□°）╯︵ ┻━┻</li>
+        <li>┻━┻︵ \(°□°)/ ︵ ┻━┻</li>
```
Comment (mattbasta) [inline popup.html]: this is going to cause the button to wrap

File: popup.html [hunk 2]
```diff
@@ -19,10 +21,14 @@
...
         <li>¯\_(ツ)_/¯</li>
         <li>( ͡° ͜ʖ ͡°)</li>
+        <li>༼ຈل͜ຈ༽</li>
         <li>ಥ_ಥ</li>
         <li>ヽ༼ ಠ益ಠ ༽ﾉ</li>
         <li>щ(ಥДಥщ)</li>
         <li>ლ(ಠ益ಠ)ლ</li>
+        <li>╭∩╮（︶︿︶）╭∩╮</li>
+        <li>☜(ﾟヮﾟ☜)</li>
+        <li>Ỏ̷͖͈̞̩͎̻̫̫̜͉̠̫͕̭̭̫̫̹̗̹͈̼̠̖͍͚̥͈̮̼͕̠̤̯̻̥̬̗̼̳̤̳̬̪̹͚̞̼̠͕̼̠̦͚̫͔̯̹͉͉̘͎͕̼̣̝͙̱̟̹̩̟̳̦̭͉̮̖̭̣̣̞̙̗̜̺̭̻̥͚͙̝̦̲̱͉͖͉̰̦͎̫̣̼͎͍̠̮͓̹̹͉̤̰̗̙͕͇͔̱͕̭͈̳̗̭͔̘̖̺̮̜̠͖̘͓̳͕̟̠̱̫̤͓͔̘̰̲͙͍͇̙͎̣̼̗̖͙̯͉̠̟͈͍͕̪͓̝̩̦̖̹̼̠̘̮͚̟͉̺̜͍͓̯̳̱̻͕̣̳͉̻̭̭̱͍̪̩̭̺͕̺̼̥̪͖̦̟͎̻̰_Ỏ̷͖͈̞̩͎̻̫̫̜͉̠̫͕̭̭̫̫̹̗̹͈̼̠̖͍͚̥͈̮̼͕̠̤̯̻̥̬̗̼̳̤̳̬̪̹͚̞̼̠͕̼̠̦͚̫͔̯̹͉͉̘͎͕̼̣̝͙̱̟̹̩̟̳̦̭͉̮̖̭̣̣̞̙̗̜̺̭̻̥͚͙̝̦̲̱͉͖͉̰̦͎̫̣̼͎͍̠̮͓̹̹͉̤̰̗̙͕͇͔̱͕̭͈̳̗̭͔̘̖̺̮̜̠͖̘͓̳͕̟̠̱̫̤͓͔̘̰̲͙͍͇̙͎̣̼̗̖͙̯͉̠̟͈͍͕̪͓̝̩̦̖̹̼̠̘̮͚̟͉̺̜͍͓̯̳̱̻͕̣̳͉̻̭̭̱͍̪̩̭̺͕̺̼̥̪͖̦̟͎̻̰</li>
```
Comment (mattbasta) [inline popup.html]: this is just going to be bad

Comment (knewkirk) [inline popup.html]: awww come on, it'll be fun!

File: popup.html (hunk 1 above)
Comment (knewkirk) [inline popup.html]: overflow: hidden? :-D

File: popup.html (hunk 2 above)
Comment (mattbasta) [inline popup.html]: post-25055-nope-not-doing-that-gif-s3vn

File: popup.html (hunk 1 above)
Comment (mattbasta) [inline popup.html]: its-not-going-to-happen-mean-girls

**Pull request closed without merging**